app/pb_kits/playbook/pb_badge/docs/_badge_colors.html.erb

```diff
@@ -91,7 +91,6 @@
 <div>
   <%= pb_rails("badge", props: {
     text: "+1",
-    variant: "neutral",
```

Comment:
Why delete this? To expose that it is the default and thus does not need to be declared.